pkg/network/ebpf/c/protocols/tls/https.h

```diff
@@ -55,14 +56,21 @@ static __always_inline void classify_decrypted_payload(protocol_stack_t *stack,
     // Protocol is not HTTP/HTTP2/gRPC
     if (is_amqp(buffer, len)) {
         proto = PROTOCOL_AMQP;
+        goto update_stack;
+    }
+
+    if (is_mysql(t, buffer, len)) {
+        log_debug("TLS: MySQL detected");
+        proto = PROTOCOL_MYSQL;
+        goto update_stack;
     }
```

Comment:
does it make sense to use `goto` instead of `if-else`?
I'd say - check complexity of the program, and use the leaner solution